### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by yashjain28**

### CODE REVIEW

File: system.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "auth": true,
-    "description": "",
-    "messaging_url": "platform.clearblade.com:1883",
-    "name": "PackageDev_jStat",
+    "description": "Statistical Analysis Toolkit Library for ClearBlade Platform",
+    "messaging_url": "staging.clearblade.com:1883",
+    "name": "dev-jstat-statistical-analysis",
```
Comment (rreinold) [inline system.json]: I recommend changing this value to 'jstat-statistical-analysis'

File: portals/JStatStatistics.json [hunk 2]
```diff
@@ -786,7 +723,7 @@
                         "id": "19a1a7fe-70d6-44ec-ac3a-752afddac5cc",
                         "incoming_parser": {
                             "isDebugOn": false,
-                            "value": "return this.datasource.map(function(e){\n    return e.data\n})"
+                            "value": "debugger\nreturn this.datasource.map(function(e){\n    return e.data\n})"
```
Comment (rreinold) [inline portals/JStatStatistics.json]: rm debugger

File: code/services/JStatExampleNormalDeviate/JStatExampleNormalDeviate.js [hunk 3]
```diff
@@ -1,5 +1,9 @@
-Y_OFFSET = 2
-function Example_NormalDeviate(req, resp){
+/**
+ * This example creates a normal deviate dataset
```
Comment (rreinold) [inline code/services/JStatExampleNormalDeviate/JStatExampleNormalDeviate.js]: link to that API Method

**Review by rreinold — CHANGES_REQUESTED** [review]

File: system.json (hunk 1 above)
Comment (yashjain28) [reply system.json]: 60c64ee9a362a0d40a1dbe84bd0ed466450ae98c

File: portals/JStatStatistics.json (hunk 2 above)
Comment (yashjain28) [reply portals/JStatStatistics.json]: 60c64ee9a362a0d40a1dbe84bd0ed466450ae98c

File: code/services/JStatExampleNormalDeviate/JStatExampleNormalDeviate.js (hunk 3 above)
Comment (yashjain28) [reply code/services/JStatExampleNormalDeviate/JStatExampleNormalDeviate.js]: https://github.com/rreinold/jstat-statistics-toolkit/pull/1/commits/60c64ee9a362a0d40a1dbe84bd0ed466450ae98c

**Pull request merged**